tools/st2ctl

```diff
@@ -233,10 +233,19 @@ function getpids(){
       echo "${COM} is not running"
       if [ "$COM" == "st2sensorcontainer" ] || [ "$COM" == "sensor_container" ]
       then
-        echo "Sensor container did not detect valid sensors to run."
-        echo "Please install sensors or check sensors configuration for existing ones."
-        echo "See http://docs.stackstorm.com/sensors.html."
-        echo "Use 'st2ctl restart-component st2sensorcontainer' to restart after you fix the problems."
+        msg="
+  Sensor container service was unable to load any sensors to load in
+  the 'base_pack' directory (or whatever path this needs to be). This may
```

Comment:
I'd drop the `or whatever path this needs to be`. I only meant that for conversational purposes